media/js/firefox/releasenotes.js

```diff
@@ -76,4 +64,4 @@
 
     initSubMenu();
 
-})(window.jQuery, window.Mozilla, window.Waypoint);
+})(window.jQuery, window.Mozilla);
```

Comment:
not a blocker-- but I know in other places we are removing jQuery. Do you think it's worth removing it from this file? 🤷‍♀️ or maybe it doesn't matter because this page is being ported to protocol?

Reply:
I think all of this will be replaced by Protocol js in the port :)